OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+
+## 3.6. Docker Security Configuration
+
+### A. Configuring auditing rules for docker 
+
+Add the following statements to the audit rules file - `/etc/audit/rules.d/audit.rules`
+
+```
+-w /usr/bin/docker -p wa
+-w /var/lib/docker -p wa
+-w /etc/docker -p wa
+-w /lib/systemd/system/docker.service -p wa
+-w /lib/systemd/system/docker.socket -p wa
+-w /etc/default/docker -p wa
+-w /etc/docker/daemon.json -p wa
+-w /usr/bin/docker-containerd -p wa
+-w /usr/bin/docker-runc -p w
+```
+Restart Audit Daemon using below commands
+```
+sudo systemctl restart auditd
+ ```
+### B. Configuring Docker daemon
+
+Add the following statements to the docker daemon json file - `/etc/docker/daemon.json`
+
+Note: For configuring centralized logging using syslog driver, syslog server details need to be configured. 
+
+Further details can be found at https://docs.docker.com/config/containers/logging/configure/
+```
+{
+    "icc": false,
+    "log-driver": "syslog",
+    "log-opts": {
+        "syslog-address": "Protocol://IpAdressofSysLogserver:port"
+               },
+    "live-restore": true,
+    "userland-proxy": false,
+    "no-new-privileges": true
+}
+```
+Restart docker daemon 
+```
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+sudo systemctl show --property Environment docker
+```
+# 4. Building Oracle WebCenter Content Docker Image
+
+To build/use a WebCenter Content image you should have Oracle FMW Infrastrucure image.
+
+## 4.1. Pulling Oracle FMW Infrastructure Docker Install Image
+
+Get Oracle FMWInfrastructure Image -
+
+Pulling Image from container-registry.oracle.com
+Refer Documentation here: https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure
+```
+docker login container-registry.oracle.com
+
+docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+docker tag  container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4 oracle/fmw-infrastructure:12.2.1.4.0
+docker rmi container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+
+```
+Note: If you are pulling the images from the container-registry.oracle.com, then follow the steps in this wiki page - https://docs.oracle.com/cd/E37670_01/E75728/html/oracle-registry-server.html.
+You first need to login to https://container-registry.oracle.com/ site in a browser and Accept the Terms and Conditions, then you can pull the images from the container-registry.oracle.com
+
+Alternatively to build this image yourself, please refer README.md under docker/OracleFMWInfrastructure for details on how to build Oracle FMW Infrastructure image.
+
+## 4.2. Building Docker Image for WebCenter Content
+
+IMPORTANT: You have to download the binary for WebCenter Content shiphome and put it in place (see Checksum file inside dockerfiles/). Set the Proxies in the environment before building the image as required.
```

Comment:
Please link to the checksum file and provide the command required to verify the checksum (and output).